govcd/api_vcd_versions.go

```diff
@@ -46,7 +46,8 @@ var apiVersionToVcdVersion = map[string]string{
 	"33.0": "10.0",
 	"34.0": "10.1",
 	"35.0": "10.2",
-	"36.0": "10.3", // Provisional version for non-GA release. It may change later
+	"36.0": "10.3",
+	"37.0": "10.4", // Provisional version for non-GA release. It may change later
```

Comment:
Not sure if I need to change some other file.

Reply:
I think nothing more should be needed. In fact, these mappings were mainly meant for XML based APIs and custom version IIRC. 

OpenAPI endpoints rely on `endpointElevatedApiVersions` (which you already updated)